### DISCUSSION TITLE
Training SiamRPN++ with Resnet-18

### DISCUSSION THREAD

**Discussion by sjtuchenye** [STVIR/pysot#401] [General]
Sorry I can not find the pretrained Resnet-18 backbone in this repo. Would you please release your pretrained Resnet-18? Or it is necessary to use pretrained backbone when using Resnet-18?
Thank you.

**Comment by ZhiyuanChen** [comment]
I think it's very unlikely for us to provide a pretrained model for resnet-18. I would suggest you to try start from scratch or start from imagenet pretrained resnet-18 and see how they work

**Comment by ZhiyuanChen** [comment]
Since no further question is asked, I'm closing this issue for now.
Please create a new issue should you have other questions.
